src/pages/OrderPage.tsx

```diff
@@ -0,0 +1,36 @@
+import Nothing from '../components/common/Nothing';
+import Spinner from '../components/common/Spinner';
+import Title from '../components/common/Title';
+import OrderItemList from '../components/orderList/OrderItemList';
+import MainLayout from '../components/PageMainLayout';
+import { IMAGE_PATH } from '../constants';
+import { useGetOrderList } from '../hooks/useGetOrderList';
+
+const OrderPage = () => {
+  const { orderList, isLoading } = useGetOrderList();
+
+  if (isLoading) return <Spinner />;
```

Comment:
```suggestion
  if (isLoading) return <MainLayout><Spinner /></MainLayout>;
```
이렇게 해야 렌더링 과정이 더 자연스럽지 않을까 싶기도 하네용